ibis/backends/snowflake/compiler.py

```diff
@@ -164,6 +164,9 @@ def visit_Literal(self, op, *, value, dtype):
             return sge.HexString(this=value.hex())
         return super().visit_Literal(op, value=value, dtype=dtype)
 
+    def visit_Arbitrary(self, op, *, arg, where):
+        return self.f.get(self.agg.array_agg(arg, where=where), 0)
```

Comment:
Unfortunately snowflake's `ANY_VALUE` doesn't have the same standard null-skipping behavior that other backends' implementations do. Since snowflake is the one out of compliance I decided that enforcing returning a non-null value (if possible) was more important than relying on the simpler `ANY_VALUE` builtin.

Reply:
Hm, would `min` work instead of building a potentially large array?

Reply:
This is what we do for compiling `First` in snowflake, just mimicked it here.  I was thinking about implementing this as `min` for all other backends in a followup since that's till an arbitrary row if you think that'd make more sense.

Reply:
Oh, this is pre-existing. Nevermind!